docs/v1.md

```diff
@@ -23,6 +23,7 @@
 - protocol(): `string`
 - version(): `string`
 - value(): `any`
+- extensions(): `Extensions`
```

Comment:
Shouldnt we remove `extensions()` from `Bindings` then? Not sure whats the use case